test/bigtest/tests/jobs-filter-test.js

```diff
@@ -0,0 +1,163 @@
+
+    beforeEach(async function () {
+      this.server.createList('job', endedJobsAmount, { status: { value: 'ended', label: 'Ended' } });
+      this.server.createList('job', queuedJobsAmount, { status: { value: 'queued', label: 'Queued' } });
+      this.server.createList('job', inProgressJobsAmount, { status: { value: 'in_progress', label: 'In progress' } });
+      this.visit('/local-kb-admin');
+      await jobs.runningStatusCheckbox.clickQueuedJobCheckbox();
+      await jobs.runningStatusCheckbox.clickInProgressJobCheckbox();
+      await jobs.whenInstancesArePresent(totalJobsAmount);
+    });
+
+    describe('filtering by', () => {
+      beforeEach(async function () {
+        await jobs.runningStatusCheckbox.clickInProgressJobCheckbox();
+      });
+
+      it('in progress jobs should show all the in progress jobs', () => {
+        expect(jobs.instanceList.size).to.equal(inProgressJobsAmount);
+      });
+    });
+
+    describe('queued jobs', () => {
+      beforeEach(async function () {
+        await jobs.clickResetAll();
+        await jobs.runningStatusCheckbox.clickInProgressJobCheckbox();
+      });
+
+      it('should show all the queued jobs', () => {
+        expect(jobs.instanceList.size).to.equal(queuedJobsAmount);
+      });
+    });
+
+    describe('in progress, queued and ended jobs', () => {
+      beforeEach(async function () {
+        await jobs.clickResetAll();
+        await jobs.runningStatusCheckbox.clickEndedJobCheckbox();
+      });
+
+      it('should show all the jobs', () => {
+        expect(jobs.instanceList.size).to.equal(totalJobsAmount);
+      });
+    });
+  });
+
+  describe('result filter tests', () => {
+    const successJobsAmount = 2;
+    const partialSuccessJobsAmount = 3;
+    const failureJobsAmount = 4;
+    const interruptedJobsAmount = 5;
+    const totalJobsAmount = successJobsAmount + partialSuccessJobsAmount + failureJobsAmount + interruptedJobsAmount;
+
+    beforeEach(async function () {
+      this.server.createList('job', successJobsAmount, { result: { value: 'Success', label: 'success' } });
+      this.server.createList('job', partialSuccessJobsAmount, { result: { value: 'Partial success', label: 'Partial success' } });
+      this.server.createList('job', failureJobsAmount, { result: { value: 'Failure', label: 'Failure' } });
+      this.server.createList('job', interruptedJobsAmount, { result: { value: 'Interrupted', label: 'Interrupted' } });
+      await this.visit('/local-kb-admin');
+      await jobs.runningStatusCheckbox.clickQueuedJobCheckbox();
+      await jobs.runningStatusCheckbox.clickInProgressJobCheckbox();
+      await jobs.whenInstancesArePresent(totalJobsAmount);
+    });
+
+    describe('filtering by', () => {
+      beforeEach(async function () {
+        await jobs.resultCheckbox.clickSuccessResultCheckbox();
+      });
+
+      it('success filter should show all successful jobs', () => {
```

Comment:
This case isn't passing for me with `AssertionError: expected 13 to equal 2`.

I wonder whether lines 73-76 should have different values for `value` and `label`? At the very least, `label` in L73 seems like it should be `Success` and not `success`.